lib/signs_ui/ueberauth/strategy/fake.ex

```diff
@@ -0,0 +1,38 @@
+defmodule SignsUi.Ueberauth.Strategy.Fake do
+  use Ueberauth.Strategy
+
+  def handle_request!(conn) do
```

Comment:
I think if you mark these as `@impl Ueberauth.Strategy` then you'll get compile time warnings if you don't implement all the functions.

Reply:
Done